polyapi/utils.py

```diff
@@ -261,13 +262,22 @@ def rewrite_arg_name(s: str):
 
 
 def is_valid_polyapi_url(_url: str):
+    # in dev allow localhost (and 127.0.0.1) over http *or* https
+    if os.getenv("PYTHON_ENV") == "development":
+        parsed = urlparse(_url)
+        if parsed.scheme in ("http", "https") and parsed.hostname in ("localhost", "127.0.0.1"):
+            return True
```

Comment:
I don't think we even need `PYTHON_ENV=development` check here. Just always accept localhost/127.0.0.1?

Reply:
we actually do need to pass in NODE_ENV=development for the js client. Maybe keep it consistent?